rpcs3/config.h

```diff
@@ -180,9 +183,6 @@ namespace convert
 			if (value == "59.94")
 				return rsx_frame_limit::_59_94;
 
-			if (value == "30")
-				return rsx_frame_limit::_30;
-
 			if (value == "60")
 				return rsx_frame_limit::_60;
 
```

Comment:
hmm, why did you change the location of the conditional check ? also the location of  _30 in the enum class ?